test/unit_tests/braket/experimental/autoqasm/test_api.py

```diff
@@ -865,16 +916,16 @@ def empty_program() -> None:
         pass
 
     expected = """OPENQASM 3.0;"""
-    assert empty_program().to_ir() == expected
+    assert empty_program.to_ir() == expected
 
 
 def test_main_return():
-    @aq.main
-    def main() -> int:
-        return 1
-
+    # TODO: outputs?
```

Comment:
bump -- reminder to address this

Reply:
I believe this is out of scope for this PR. I believe it's on our radar already, so would you prefer I remove this todo?